javascript/builtins/Intl/NumberFormat.json

```
@@ -527,7 +527,9 @@
                   "nodejs": {
                     "version_added": false
                   },
-                  "opera": "mirror",
+                  "opera": {
+                    "version_added": false
+                  },
```

Opera should still mirror its data from Chrome; I would be VERY surprised if a core JS feature like this was intentionally disabled in Opera.
```suggestion
                  "opera": "mirror",
```

---

You're absolutely right, my apologies. Must've screwed up my initial testing somehow.